tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/remote/RemoteInfinispanTestSupport.java

```diff
@@ -54,78 +53,17 @@
     private static final Logger INFINISPAN_LOG =
         LoggerFactory.getLogger("org.eclipse.jetty.server.session.remote.infinispanLogs");
 
-    static GenericContainer infinispan;
+    private static String IMAGE_NAME = System.getProperty("infinispan.docker.image.name", "infinispan/server") +
+            ":" + System.getProperty("infinispan.docker.image.version", "11.0.9.Final");
 
-    static
-    {
-        try
-        {
-            //Testcontainers.exposeHostPorts(11222);
-            long start = System.currentTimeMillis();
-            String infinispanVersion = System.getProperty("infinispan.docker.image.version", "11.0.9.Final");
-            infinispan =
-                new GenericContainer(System.getProperty("infinispan.docker.image.name", "infinispan/server") +
-                    ":" + infinispanVersion)
-                .withEnv("USER", "theuser")
+    private static final GenericContainer infinispan = new GenericContainer(IMAGE_NAME)
```

Comment:
```suggestion
    private static final GenericContainer __infinispan = new GenericContainer(IMAGE_NAME)
```

Reply:
as it static final I would prefer `INFINISPAN`

Reply:
Sure, all uppercase is fine if it's `final`.

Reply:
fixed